sdcm/results_analyze/__init__.py

```diff
@@ -729,18 +729,18 @@ def check_regression(self, test_id, is_gce=False, email_subject_postfix=None,
         # get test res
         doc = self.get_test_by_id(test_id)
         if not doc:
-            self.log.error('Cannot find test by id: {}!'.format(test_id))
-            return False
+            raise ValueError(f'Cannot find test by id: {test_id}')
         self.log.debug(PP.pformat(doc))
 
         test_stats = self._test_stats(doc)
         if not test_stats:
-            return False
+            raise ValueError(f'Cannot find test by id: {doc.get("_id", "unknown test")}')
```

Comment:
what is the impact when raising instead of returning False?

Reply:
I want every failure to be raise as event 

With return false it's losing some part of the context